pyramid_zipkin/tween.py

```diff
@@ -60,6 +62,14 @@ def _get_settings_from_request(request):
             " and a message as params and logs the message via scribe/kafka."
         )
 
+    context_stack = None
+    if 'request_context' in settings:
+        request_context = getattr(request, settings['request_context'], None)
+        context_stack = getattr(request_context, 'zipkin_context', None)
```

Comment:
As discussed offline: let's try out supporting dot syntax for the setting, so that we don't have to hardcode the zipkin_context attribute here.

Reply:
Just a reminder to address this.